SchedulerJobs/SchedulerJobs.Services/ELinksService.cs

```diff
@@ -31,11 +32,12 @@ public ELinksService(IPeoplesClient peoplesClient, ILeaversClient leaversClient,
 
         public async Task ImportJudiciaryPeopleAsync(DateTime fromDate)
         {
+            var invalidPeoplePersonalCode = new List<string>();
             var currentPage = 1;
 
-            try
+            while (true)
```

Comment:
have we taken consideration about infinite loop here? Or are we confident that peopleresult list will eventually be zero? Although I can see this isn't a new change however we have changed the order of the try catch and while blocks here, does our unit test check of the list being null or empty as I cant see any other way this code will ever break?